x/tokenfactory/client/cli/tx.go

```diff
@@ -46,7 +46,7 @@ func NewCreateDenomCmd() *cobra.Command {
 				return err
 			}
 
-			txf := tx.NewFactoryCLI(clientCtx, cmd.Flags()).WithTxConfig(clientCtx.TxConfig).WithAccountRetriever(clientCtx.AccountRetriever)
+			txf, err := tx.NewFactoryCLI(clientCtx, cmd.Flags())
```

Comment:
error unchecked

Reply:
Any update on this?

Reply:
will add a commit on this, i've been caught up with trying to pass the last testcase

Reply:
done